### ISSUE TITLE
[Question:] Why does the header loses its collapse-state on setState

### ISSUE THREAD

**Issue by ehhc** [aryzhov/flutter-expandable#3]
Hi,
I have the following app in which the default example is wrapped in an expandable-panel. Initially the panel is collapsed but i can expand it by clicking the header.
Now everytime i press the increment button, the panel gets collapsed again. How can i prevent that?

```
import 'package:flutter/material.dart';
import 'package:expandable/expandable.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key, this.title}) : super(key: key);
  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: ExpandablePanel(
            initialExpanded: false,
            header: Text("Header"),
            expanded: Text('You have pushed the button this many times: $_counter')
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: _incrementCounter,
        tooltip: 'Increment',
        child: Icon(Icons.add),
      ), // This trailing comma makes auto-formatting nicer for build methods.
    );
  }
}

```

**Comment by StefanLobbenmeier** [comment]
Hi :) I had the same issue, so I created a pull request for this issue. I @aryzhov will pull this, but if you are impatient you can reference my repository in your pubspec.yaml like this:

```
dependencies:
  flutter:
    sdk: flutter
  expandable:
    git: https://github.com/StefanLobbenmeier/flutter-expandable
```

**Comment by aryzhov** [comment]
@StefanLobbenmeier Thank you for working on this. I'll review your changes today. I wasn't getting notifications on the issues somehow, so I missed this and other questions. I'll catch up.

**Comment by aryzhov** [comment]
@ehhc  I just published version 2.1.0 that contains a fix for this issue. Thanks @StefanLobbenmeier.

**Comment by ehhc** [comment]
@StefanLobbenmeier @aryzhov thanks :)
